### PR TITLE
docs: fix typo in rbac docs

### PR DESCRIPTION
**Pull request by hamidzr**
## Checklist

- [ ] Changes have been manually QA'd
- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

### CODE REVIEW

File: docs/cluster-setup-guide/security/rbac.rst [hunk 1]
```diff
@@ -408,7 +408,7 @@ projects and experiments within its scope.
 ``WorkspaceAdmin`` role supersedes the ``Editor`` role, and includes permissions to edit or delete
 workspaces, and modify role assignments within its scope.
 
-Users who posess this role on a particular workspace can assign roles to other users on this
+Users who take this role on a particular workspace can assign roles to other users on this
```
Comment (hamidzr) [inline docs/cluster-setup-guide/security/rbac.rst:411]: take on?

**Comment by keita-determined** [comment]
@hamidzr 
i think you need to assign someone instead of @determined-ai/docs cuz there's no one in there

**Review by keita-determined — APPROVED** [review]
LGTM. Prob `take on`? whichever you want

**Comment by hamidzr** [comment]
> LGTM. Prob `take on`? whichever you want

thanks for the review. If what I have is correct I'll just go with that. I'm updating some of this soon in another rbac project.

**Pull request merged**